docs/parkiNarodowe.js

```diff
@@ -0,0 +1,177 @@
+
+function getLocalStgOr(key, default_value) {
+    x = localStorage.getItem(key);
+    if (x === null) return default_value;
+    else return Number(x);
+}
+
+function saveXPAndLvl(xp, lvl) {
+    localStorage.setItem("xp", xp);
+    localStorage.setItem("lvl", lvl);
+}
+
+window.onload = function () {
+    let experience = getLocalStgOr("xp", 0);
+    let lvl = getLocalStgOr("lvl", 1);
+    let curFibonachi = 2 + lvl;
+    let expCap = fibonachiNumber(curFibonachi);
+
+    const lvlBar = document.querySelector("#myBar");
+    const lvlText = document.querySelector("#overlayText");
+
+    function fibonachiNumber(point) {
+        if (point == 0) {
+            return 0;
+        }
+        else if (point == 2 || point == 1) {
+            return 1;
+        }
+        else {
+            let loop = 1;
+            let a = 1;
+            let b = 1;
+            while (loop < point) {
+                if (loop % 2 != 0) {
+                    a = a + b;
+                }
+                else {
+                    b = a + b;
+                }
+                loop += 1;
+            }
+            if (a > b) {
+                return a;
+            }
+            return b;
+        }
+    }
+
+    function moveBar(curAmmount, maxAmmount) {
+        var width = Math.ceil(curAmmount / maxAmmount * 100);
+        lvlBar.style.width = width + "%";
+    }
+
+    moveBar(experience, expCap);
+    lvlText.innerText = experience + "/" + expCap + " XP, level " + lvl;
+    function expUp() {
+        experience += 1;
+        updateXP();
+    }
+    function updateXP() {
+        if (experience >= expCap) {
+            lvl += 1;
+            curFibonachi += 1;
+            expCap = fibonachiNumber(curFibonachi);
+            experience = 0;
+            if (expCap <= 0) {
+                console.log("Wow. Int overload... that was so unexpected... If your testing this library, then I understand how you got here, else... go touch some grass, pls");
+            }
+        }
+
+        saveXPAndLvl(experience, lvl);
+
+        moveBar(experience, expCap);
+        lvlText.innerText = experience + "/" + expCap + " XP, level " + lvl;
+    }
+
+    // Get the HTML Canvas
+    const canvas = document.querySelector("#image_canvas");
+    // Get the canvas context
+    const ctx = canvas.getContext('2d');
+
+    const name = document.querySelector('#name');
+    const points = document.querySelector('#points');
+
+    const chosen = document.querySelector('#chosen');
+
+    // Set this variable to any image source
+    const IMAGE_SRC = 'parkiNarodoweRaw.png';
+
+    // Create a new image and add it to your canvas
+    const img = new Image();
+    img.height = 700;
+    img.width = 800;
+    // Wait for the image to load before drawing to the canvas
+    img.addEventListener('load', function () {
+        ctx.drawImage(img, 0, 0);
+    }, false);
+
+    img.crossOrigin = '';
+    img.src = IMAGE_SRC;
+
+    // This is important to avoid cross origin errors
```

Comment:
What is important to avoid them?

Reply:
well... you see the cross origin value in img? It somehow traveled behind the comment